docs/LicenseInfo.md

```diff
@@ -0,0 +1,17 @@
+## License info
+
```

Comment:
Let's have two sections:

#Licence info 
## Community version
... let's mention that all code available on github and https://hub.docker.com/r/touk/nussknacker is included 
## Enterprise version
...

Reply:
applied this